### ISSUE TITLE
AI responses to new quiz card format

### ISSUE THREAD

**Issue by esocha13** [Bgorson/giftology#135]
I know we don't have card side B figured out yet, but start transitioning the production AI responses to our new card format

**Comment by Bgorson** [comment]
So what would this format look like.. we don't have any logic to 'favorite' AI items or flip the card at this point.

**Comment by esocha13** [comment]
Presumably getting the wishlist working for GDP products is very challenging? Assuming that's the case, can the wishlist buttons be eliminated for GDP products and Card Side B populated with the Name and purchase price, since you already have that pulled in?

This is part of why I was looking at the Amazon API glossary of attributes. What I'd really like is to include the attributes CustomerReviews.Count and CustomerReview.Rating from the Amazon API on cardside B, is that possible before you get out of town?

**Comment by Bgorson** [comment]
Yes, our favorites stores a list of IDs that correspond to products.. saving the chatGPT ones would be a very different approach/pattern. 
And yea I think I can get those in

**Comment by esocha13** [comment]
Hypothetically, once we hit the Amazon API could pull down an Amazon ASIN and save that for the wishlist? The trick is just knowing which products have an ASIN and need the API vs which ones are from our database.

**Comment by Bgorson** [comment]
Yep. Exactly. Issue isn't that it can't be done. Just deviates from our current pattern.

**Comment by esocha13** [comment]
Can save that for later, I just want a quick path to having the cards for now

**Comment by Bgorson** [comment]
So I don't think I can get this safely up and deployed tonight. Lets check in on this Monday to map out exactly how we want these to look and where the data is coming from
